MonoGame.Framework/Media/MediaPlayer.cs

```diff
@@ -69,17 +112,41 @@ private set
                     EventHelpers.Raise(null, MediaStateChanged, EventArgs.Empty);
                 }
             }
-        }
-
+        }
+
+        /// <summary>
+        /// Determines whether the game has control of the background music.
+        /// </summary>
+        /// <remarks>
+        /// If a gamer is currently playing their own music as the background to your game,
+        /// calls to <see cref="Play(Song)"/>, <see cref="Stop()"/>, <see cref="Pause()"/>,
+        /// <see cref="Resume()"/>, <see cref="MoveNext()"/>, and <see cref="MovePrevious"/>
+        /// might have no effect, depending on the platform.
+        /// If another application's background music is playing, your game will need to call
```

Comment:
I think this sentence is missing a word

`If another application's background music is playing, your game will need to call <???> in order to pause the other application's background music in order to play the game's music.`

Reply:
It looks like it, but that's exactly how this sentence is written in XNA docs. Open XNA docs [doesn't mention this](https://learn.microsoft.com/en-us/previous-versions/windows/xna/dd254741(v=xnagamestudio.30)#remarks) at all. 
I'll probably remove this part for now

Reply:
I guess is was `Pause` based on the context. But if we don't know how this works on each platform its probably best not to mention it. 👍